torchgeo/models/rcf.py

```diff
@@ -104,3 +107,111 @@ def forward(self, x: Tensor) -> Tensor:
             assert len(x1a.shape) == 2
             output = torch.cat((x1a, x1b), dim=1)
             return output
+
+
+class MOSAIKS(RCF):
+    """This model extracts MOSAIKS features from its input.
+
+    MOSAIKS features are described in Multi-task Observation using Satellite Imagery &
+    Kitchen Sinks https://www.nature.com/articles/s41467-021-24638-z. Briefly, this
+    model is instantiated with a dataset, samples patches from the dataset, ZCA whitens
+    the patches, then uses those as convolutional filters to extract features with.
+
+    .. note::
+
+        This Module is *not* trainable. It is only used as a feature extractor.
+    """
+
+    def _normalize(
+        self,
+        patches: "np.typing.NDArray[np.float32]",
+        min_divisor: float = 1e-8,
+        zca_bias: float = 0.001,
+    ) -> "np.typing.NDArray[np.float32]":
+        """Does ZCA whitening on a set of input patches.
+
+        Copied from https://github.com/Global-Policy-Lab/mosaiks-paper/blob/7efb09ed455505562d6bb04c2aaa242ef59f0a82/code/mosaiks/featurization.py#L120
```

Comment:
Pretty sure https://github.com/Global-Policy-Lab/mosaiks-paper#use-of-code-and-data lets us do this but I'm no legal expert